### PR TITLE
cleaning-service-page

### PR DESCRIPTION
**Pull request by gigawave07**
Please help me review, ty.

### CODE REVIEW

**Review by gigawave07 — COMMENTED** [review]
Fix space for better visual.

**Comment by Minhnguyendhth8b** [comment]
Please fix conflict file first.

**Comment by gigawave07** [comment]
> Please fix conflict file first.

Done

File: styles.css [hunk 1]
```diff
@@ -1 +1,70 @@
+html {
+    background-color: pink;
+    
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
 
+.container {
+    margin: 100px 100px 100px 100px;
```
Comment (Minhnguyendhth8b) [inline styles.css]: Just use: margin: 100px;
if you have the same value for angle.

File: styles.css [hunk 2]
```diff
@@ -1 +1,70 @@
...
+    background-color: white;
+    background: url(background.jpg) no-repeat;
+    background-size: 1280px 768px;
+    width: 1280px;
```
Comment (Minhnguyendhth8b) [inline styles.css:11]: Shouldn't use the static width and height. It will make responsive is hard to use in the future.

File: styles.css [hunk 3]
```diff
@@ -1 +1,70 @@
...
+    align-items: center;
+}
+
+
+.navbaritem {
+    margin: 50px 40px 0px 40px;
+    color: black;
+}
+
+#LOGIN {
```
Comment (Minhnguyendhth8b) [inline styles.css]: DO NOT use the uppercase in ID or class name

File: styles.css [hunk 4]
```diff
@@ -1 +1,70 @@
...
+    margin: 0px 0px 50px 0px;
+}
+
+#signup {
+    background-color: turquoise;
+    color: white;
+    border: none;
+    width: 100px;
+    height: 50px;
+    padding: 5px 20px 5px 20px;
```
Comment (Minhnguyendhth8b) [inline styles.css]: Fix this one also in short way.

File: cleaning-service.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        <link rel="stylesheet" href="styles.css">
+        <script src="scripts.js"></script>
+        <title>Cleaning Service</title>
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <nav class="navbar">
+                    <a href=#About-us><div class="navbaritem">ABOUT US</div></a>
+                    <a href="Services"><div class="navbaritem">SERVICES</div></a>
```
Comment (Minhnguyendhth8b) [inline cleaning-service.html]: Why this one doesn't have the "# "

File: cleaning-service.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+                    <a href="Blog"><div class="navbaritem">BLOG</div></a>
+                    <a href="FAQ"><div class="navbaritem">FAQ</div></a>
+                    <div class="navbaritem"><button id="LOGIN"><b>LOGIN</b></button></div>
```
Comment (Minhnguyendhth8b) [inline cleaning-service.html]: If class name have a long word , 2 words perhaps you need to add dash into class name like this : navbar-item

Comment (Minhnguyendhth8b) [inline cleaning-service.html]: Shouldn't use b tag in button because it will make the code is hard to read.

**Review by Minhnguyendhth8b — COMMENTED** [review]
Please check all errors and optimize if availiable.

File: styles.css (hunk 3 above)
Comment (gigawave07) [reply styles.css]: Oh ok this is an important one.

**Pull request merged**